pkg/clients/aws/route53.go

```diff
@@ -52,63 +52,6 @@ type awsClient struct {
 	client route53iface.Route53API
 }
 
-// searchForHostedZone finds a hostedzone when given an aws client and a domain string
```

Comment:
@dustman9000 these too? There are a bunch of other edits after this that I wont comment on